test/SlowTests/Issues/RavenDB-12204.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Utils;
+using Raven.Client.Documents.Smuggler;
+using Raven.Tests.Core.Utils.Entities;
+using Xunit;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_12204 : RavenTestBase
+    {
+        [Fact]
+        public async Task CanMigrateFromRavenDb()
+        {
+            var file = Path.GetTempFileName();
```

Comment:
use `NewDataPath` for this It will delete the directories/files when test will end.